nifi-nar-bundles/nifi-soap-bundle/nifi-soap-processors/src/main/java/org/apache/nifi/processors/soap/GetSOAP.java

```diff
@@ -46,34 +51,30 @@
 import org.apache.nifi.processor.ProcessorInitializationContext;
 import org.apache.nifi.processor.Relationship;
 import org.apache.nifi.processor.exception.ProcessException;
-import org.apache.nifi.processor.io.OutputStreamCallback;
 import org.apache.nifi.processor.util.StandardValidators;
-import org.apache.nifi.util.StopWatch;
+
 import java.io.IOException;
-import java.io.OutputStream;
 import java.util.*;
 
-import static org.apache.axis2.namespace.Constants.URI_SOAP12_ENV;
-
-
 @SupportsBatching
-@InputRequirement(InputRequirement.Requirement.INPUT_FORBIDDEN)
+@InputRequirement(InputRequirement.Requirement.INPUT_REQUIRED)
```

Comment:
Yes, initially I had tried INPUT_ALLOWED but saw some other weird behavior. I switched to REQUIRED to get it working and meant to go back and troubleshoot the weirdness I was seeing but forgot about it